### PR TITLE
72421 attachments table

### PR DESCRIPTION
**Pull request by janid1967**
First version of Attachment table. May be changed when integrating with blobstorage.
Main idea: Common Attachment table used for different entity types via Discriminator. For now, entities Tag and Action.

Next steps after this PR: 
- endpoint for adding Tag Attachment (without blob for now) 
- endpoint for listing all Tag Attachments 

(Do not use to much time to reflect over class ProjectAggregate/Attachment.cs which is renamed to AttachmentValue.cs here. AttachmentValue.cs is for recording av value of type attachment instead of type checkbox or number. Implementation of AttachmentValue.cs will later be extended/changed to also be integrated with a blob)

### CODE REVIEW

**Comment by janid1967** [comment]
Will come a commit with unit tests soon

File: src/Equinor.Procosys.Preservation.Domain/AggregateModels/ProjectAggregate/ActionAttachment.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Equinor.Procosys.Preservation.Domain.AggregateModels.ProjectAggregate
```
Comment (hbjorgo) [inline src/Equinor.Procosys.Preservation.Domain/AggregateModels/ProjectAggregate/ActionAttachment.cs:3]: Are the sub classes necessary? What if there's just one class - Attachment?

File: src/Equinor.Procosys.Preservation.Domain/AggregateModels/ProjectAggregate/ActionAttachment.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+{
+    public class ActionAttachment : Attachment
```
Comment (hbjorgo) [inline src/Equinor.Procosys.Preservation.Domain/AggregateModels/ProjectAggregate/ActionAttachment.cs:5]: Are the sub classes necessary? What if there's just one class - Attachment?

Comment (janid1967) [reply src/Equinor.Procosys.Preservation.Domain/AggregateModels/ProjectAggregate/ActionAttachment.cs]: You're probably right. See latest commits

Comment (janid1967) [reply src/Equinor.Procosys.Preservation.Domain/AggregateModels/ProjectAggregate/ActionAttachment.cs]: ....or not!!! When update database I now get Attachment table with both TagId and ActionId foregin key columns and both are non-nullable. This is the part we get "for free" whit abstract class Attachment, subclass for ActionAttachment and TagAttachment and and Discrimnator column

Comment (janid1967) [reply src/Equinor.Procosys.Preservation.Domain/AggregateModels/ProjectAggregate/ActionAttachment.cs]: I suggest I revert back 3 commits. OK @hbjorgo ?

**Pull request merged**